src/hooks/useIntervalReRender.js

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState, useRef } from 'react';
+
+/**
+ * Forces a re-render of a component every {interval} milliseconds.
+ *
+ * Use case is primarily refreshing to show countdowns. Use should
+ * generally be avoided.
+ * @param {number} interval
+ */
+export const useIntervalReRender = interval => {
+  const [, setState] = useState({});
+  const intervalRef = useRef();
+
+  useEffect(() => {
+    intervalRef.current = setInterval(() => setState({}), interval);
+    return () => clearInterval(intervalRef.current);
+  }, [interval]);
+};
```

Comment:
This hook is will re-render the "last download 1 min ago" component every so often which will cause a recalc the delta of the last sync time and the time now - without this, theres no re-render when time changes and if you sit on a page for 10 mins it'll just say "a few seconds ago"